pages/admin/request/create-renewal.tsx

```diff
@@ -404,28 +461,73 @@ export default function CreateRenewal() {
                         _hover={{ bg: 'secondary.criticalHover' }}
                         marginRight="20px"
                         height="48px"
-                        width="149px"
+                        width="188px"
                       >
                         <Text textStyle="button-semibold">Discard request</Text>
                       </Button>
                     </CancelCreateRequestModal>
-                    <Link href="#">
-                      <Button
-                        bg="primary"
-                        height="48px"
-                        width="180px"
-                        type="submit"
-                        loading={loading}
-                      >
-                        <Text textStyle="button-semibold">Create request</Text>
-                      </Button>
-                    </Link>
+                    <Button
+                      bg="primary"
+                      height="48px"
+                      width="180px"
+                      type="submit"
+                      loading={loading}
+                    >
+                      <Text textStyle="button-semibold">Create request</Text>
+                    </Button>
                   </Stack>
                 </Box>
               </Stack>
             </Box>
           </form>
         )}
+        {/* Footer on Permit Searcher Page*/}
+        {!onRequestPage && (
+          <Box
+            position="fixed"
+            left="0"
+            bottom="0"
+            right="0"
+            paddingY="20px"
+            paddingX="188px"
+            bgColor="white"
+            boxShadow="dark-lg"
+          >
+            <Stack direction="row" justifyContent="space-between" alignItems="center">
+              <Box />
```

Comment:
Can we remove this?

Reply:
No since, it causes positioning of left buttons to go right and get screwed up for some reason. I tried playing around with it, but could not crack it.

Reply:
Try making the `Stack` have `direction="column"` instead